lib/kafka_ex/server_0_p_8_p_2.ex

```diff
@@ -213,7 +213,7 @@ defmodule KafkaEx.Server0P8P2 do
   defp update_consumer_metadata(state),
```

Comment:
This head is still private, is that intended? it's strange for a private function to wrap a public one

Reply:
This private function is used by their own module, do it need to be changed?